app/controllers/answers_controller.rb

```diff
@@ -1,11 +1,33 @@
 class AnswersController < ApplicationController
 
-  def index
+  def create
+    question = Question.find(params[:question_id])
+    answer = question.answers.new(content: params[:answer][:content], user_id: session[:user_id])
+    if answer.save
+      # Ajax
+      redirect_to question
+    else
```

Comment:
if you're doing the same thing on either side of the if/else, why have the if/else at all?